core/src/edu/asu/c3simulator/widgets/ProductDesigns.java

```diff
@@ -0,0 +1,149 @@
+package edu.asu.c3simulator.widgets;
```

Comment:
The last file! I feel like it's Christmas.

Our team should discuss pull requests, and work towards reducing their size. 200-400 lines would be ideal, but 2000-3000 is challenging to get through, and I'm afraid the quality of this review may have suffered in some places, especially the middle.

As such, you may want another reviewer to at least go over the middle files thoroughly.